QEfficient/transformers/models/cohere/modeling_cohere.py

```diff
@@ -0,0 +1,571 @@
+# -----------------------------------------------------------------------------
+#
+# Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+#
```

Comment:
can you add a delimiter here?

---------------

Reply:
Sure